test/Unused/UtilSpec.hs

```diff
@@ -27,3 +27,9 @@ spec = parallel $
 
             groupBy pName people `shouldBe` [("Jane", [Person "Jane" 10, Person "Jane" 20]), ("John", [Person "John" 20])]
             groupBy pAge people `shouldBe` [(10, [Person "Jane" 10]), (20, [Person "Jane" 20, Person "John" 20])]
+
+    describe "stringToInt" $
```

Comment:
@gabebw since there's only one spec, it doesn't need the `do` right now - if I add any others, though, I'll be sure to include it!